sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainerTypeConfiguration.md

```diff
@@ -18,7 +18,14 @@ Sets or updates the configuration settings of a container type in SharePoint Emb
 
 ## SYNTAX
 ```powershell
-Set-SPOContainerTypeConfiguration [-ContainerTypeId <ContainerTypeId>] [-DiscoverabilityDisabled <Boolean>] [-SharingRestricted <Boolean>]
+Set-SPOContainerTypeConfiguration
+ [-ContainerTypeId <ContainerTypeId>]
```

Comment:
>[-ContainerTypeId <ContainerTypeId>] [](http://example.com/codeflow?start=1&length=36)

- I think this is a mandatory param? so it shouldn't be within the `[]`
- a few params are missing too, like `ApplicationRedirectUrl`, WhatIf, Confirm, CommonParrameters

I'd suggest you use platyPS to generate this template, which makes sure nothing is missed. [See instructions](https://dev.azure.com/onedrive/ODSP%20Wiki/_wiki/wikis/OSDP-Wiki.wiki/82618/SharePoint-Online-PowerShell-cmdlets?anchor=public-cmdlet-documentation) for that